yarn.lock

```diff
@@ -7390,6 +7390,21 @@ kolibri-constants@0.1.42:
   resolved "https://registry.yarnpkg.com/kolibri-constants/-/kolibri-constants-0.1.42.tgz#2f62a8d8b8894e5cfbd47ee6564b31018818c93f"
   integrity sha512-2hUxYnzUEfhLFJO9egVSwYW8/PKob9wLeDYfB74mtIzgQ4zy6huRj3574WetK9gREi+W1Jcm7HGPsfZIFzFvrA==
 
+"kolibri-design-system@https://github.com/AlexVelezLl/kolibri-design-system#d0ec826d6dfee3be9805fd48ecb2d707df7999cf":
```

Comment:
Looks like there's still a reference to your fork of KDS in here, would be good to clear this out of the lock file!

Reply:
Solved! @rtibbles